### PR TITLE
Add MKVToolNix.app v9.9.0

### PR DESCRIPTION
**Pull request by ponyspy**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by miccal** [comment]
How is this different from that found in [Homebrew](https://github.com/Homebrew/homebrew-core/blob/master/Formula/mkvtoolnix.rb)?

File: Casks/mkvtoolnix.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'mkvtoolnix' do
+  version '9.9.0'
+  sha256 '454cb48725cd1a536052ef10c33774332999a534e2ed14b6f86f2708ad72d3ea'
+
+  url "https://mkvtoolnix.download/macos/MKVToolNix-#{version}.dmg"
+  name 'MKVToolNix'
+  homepage 'https://mkvtoolnix.download'
+
+  app "MKVToolNix-#{version}.app"
+end
```
Comment (reitermarkus) [inline Casks/mkvtoolnix.rb]: Missing a newline here.

**Comment by ponyspy** [comment]
No deps, no build. Only binary app from authors web site.
It is easier and more convenient for the simple user.
Like me :)

File: Casks/mkvtoolnix.rb (hunk 1 above)
Comment (ponyspy) [inline Casks/mkvtoolnix.rb]: Sorry I do not understand what line is missing?
I found same [cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/ableton-live-lite.rb) and it's normal.

Comment (reitermarkus) [inline Casks/mkvtoolnix.rb]: There should be an empty line after `end`.

Comment (ponyspy) [inline Casks/mkvtoolnix.rb]: Ok. I fix it.

**Comment by miccal** [comment]
@caskroom/maintainers:
```
audit for mkvtoolnix: warning
 - possible duplicate, cask token conflicts with Homebrew core formula: https://github.com/Homebrew/homebrew-core/blob/master/Formula/mkvtoolnix.rb
```

**Comment by vitorgalvao** [comment]
I don’t have a strong opinion on this one, but perhaps we should refuse it.

We have to draw the line *somewhere*, and the deduplication effort happened for a reason. We were aware some apps might be a bit more of an inconvenience to use, but that deduplication was more important. I don’t see much request for this one (we used to have it). I’m betting it’ll become abandoned pretty quickly.

**Comment by claui** [comment]
Accepting the cask would make `brew search mkvtoolnix` yield:

```
mkvtoolnix
Caskroom/cask/mkvtoolnix
```

At that point, the user **will have to make a choice;** however, either choice [would lead to the same result](https://en.wikipedia.org/wiki/Hobson's_choice).

In RPGs, this [is considered an annoyance](http://tvtropes.org/pmwiki/pmwiki.php/Main/ButThouMust). In all other places, I consider it poor UX.

**Comment by vitorgalvao** [comment]
Good thing it’s already pretty late and I’m going to bed now. Otherwise I’d lose the rest of my day on tvtropes!

But I know I’ll check it tomorrow. I wonder if I’ll be able to restrain myself to just that page.

**Comment by miccal** [comment]
@ponyspy in the spirit of deduplication, we are rejecting this Cask.

Thank you for your contribution, though.

**Comment by ponyspy** [comment]
@miccal What about this [cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/mpv.rb) or [this](https://github.com/caskroom/homebrew-cask/blob/master/Casks/djview.rb)? Remove too? I use this programs and i install it from cask, not brew.

I don't understand where you see a duplicate... I want to program without dependencies, and as fast as possible, I thought the philosophy of cask in it.

Very strange...

**Comment by claui** [comment]
@ponyspy I see your point, and how decisions like the one at hand are prone to come across as irritating, or seemingly arbitrary. Please be aware that **no matter what, contributions like yours are appreciated a lot.** Even if a cask may ultimately end up being rejected, we are thankful for people like you who bring such edge cases to our attention.

Regarding `mpv`:

The duplication was already investigated several times \([\[1\]](https://github.com/caskroom/homebrew-cask/issues/15603#issuecomment-174220603) [\[2\]](https://github.com/Homebrew/homebrew-core/issues/2260#issue-161544435)\); it has been found that [they are not the same thing](https://github.com/caskroom/homebrew-cask/pull/17447#issuecomment-174226261) so it’s completely justified to keep both around.

Regarding `djview`:

I’m with you here; the Homebrew formula is redundant and should be deleted (it is broken anyway, at least on my machine). @vitorgalvao has in fact [brought this to Homebrew’s attention](https://github.com/Homebrew/homebrew-core/issues/2260) some time ago; the thing is that [the housework that is necessary](https://github.com/caskroom/homebrew-cask/issues/15603#issuecomment-171639596) before we delete those duplicates is [not trivial at all](https://github.com/caskroom/homebrew-cask/pull/7906) and is therefore [still in progress](https://github.com/caskroom/homebrew-cask/issues/27364).

I admit that in theory, we could make such decisions transparent to users at all times. In practice, however, I believe this is incredibly hard; the available resources and energy are certainly better spent on pushing the project forward.

No matter what, I’m looking forward to your future contributions. 👍

**Pull request closed without merging**